### PR TITLE
BMR 47, BMR 44, BMR 46 Calendar feature with home tab

### PR DESCRIPTION
**Pull request by kamdaou-amalitech**
Implemented the Home tab,
Implemented reusable composables for the calendar screen
Implemented calendar and home screens

### CODE REVIEW

**Comment by kamdaou-amalitech** [comment]
https://github.com/AmaliTech-Training-Rw/Meeting-Room-booking-App/assets/132997531/aa96b9ee-707c-4b5b-ad1d-38907d0f5e54

**Comment by esthcarelle** [comment]
@kamdaou-amalitech Good job, looks nice. Why do we have 3 tickets combined in one PR?

File: ui/home/src/main/java/com/amalitech/home/calendar/CalendarScreen.kt [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+                        backgroundColor = bookingsContentBackgroundColor
+                    )
+                    Spacer(Modifier.height(spacing.spaceSmall))
+                }
+            }
+        }
+    }
+}
+
+fun formatDate(localDate: LocalDate): String {
```
Comment (bensek) [inline ui/home/src/main/java/com/amalitech/home/calendar/CalendarScreen.kt:201]: Move this to a Util Class same with the formatTime

File: ui/home/src/main/java/com/amalitech/home/HomeScreen.kt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import com.amalitech.core_ui.theme.LocalSpacing
+import com.amalitech.home.calendar.CalendarScreen
+import com.amalitech.home.components.HomeTab
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+fun HomeScreen(
+    viewModel: HomeViewModel = koinViewModel()
+) {
+    val selectedTab by remember {
```
[the remark this answers is not in the document]
Comment (kamdaou-amalitech) [reply ui/home/src/main/java/com/amalitech/home/HomeScreen.kt]: This is a mistake from me. Please check out the new implementation

**Review by bensek — APPROVED** [review]

**Review by esthcarelle — APPROVED** [review]

**Pull request merged**